central/policy/datastore/datastore_impl.go

```diff
@@ -331,9 +332,15 @@ func (ds *datastoreImpl) ImportPolicies(ctx context.Context, importPolicies []*s
 }
 
 func (ds *datastoreImpl) importPolicy(ctx context.Context, policy *storage.Policy, overwrite bool, policyNameIDMap map[string]string) *v1.ImportPolicyResponse {
+	if policy.GetId() == "" {
```

Comment:
out of curiosity: what was the purpose behind moving this up?

Reply:
the response clone has the ID without it needing to be set